src/main/webapp/app/tasks/task-list/task-list.component.spec.ts

```diff
@@ -53,4 +54,31 @@ describe('TaskListComponent', () => {
     // then
     expect(deleteEmitter).toHaveBeenCalledWith({id: 'id', name: 'My task'});
   });
+
+  it('should filter the task using a search term', () => {
+    // given
+    component.tasks = [];
```

Comment:
Why are you not assigning the array with the values instead of pushing them into the array afterwards?